compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -100,8 +100,9 @@ data Error
   | EExpectedOptionalType  !Type
   | EEmptyCase
   | EExpectedTemplatableType !TypeConName
-  | EImportCycle           ![ModuleName]
-  | EDataTypeCycle         ![TypeConName]
+  | EImportCycle           ![ModuleName] -- Never reported
```

Comment:
Would be useful to add a `TODO` for these.

Reply:
done